### PR TITLE
feat: remove opensearch error and warning checks from optimize fe

### PR DESCRIPTION
**Pull request by OmranAbazid**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

related to #23588

### CODE REVIEW

File: optimize/client/src/components/Reports/Report.js [hunk 1]
```diff
@@ -101,11 +101,6 @@ export class Report extends React.Component {
 
   render() {
     const {report, serverError} = this.state;
-    const hasOpensearchError = serverError?.detailedMessage?.includes('No interpreter registered');
-
-    if (hasOpensearchError) {
-      return <ErrorPage text={t('apiErrors.reportNotSupportedForOpenSearch')} />;
-    }
```
Comment (OmranAbazid) [inline optimize/client/src/components/Reports/Report.js:108]: Hey @grlimacan, 
I assumed that there is always an interpreter to all types of reports and no such error should happen. 
Am i right here?

**Comment by OmranAbazid** [comment]
Hey Gergey, I assigned you the C8 change as well since they are kinda similar to the C7 change.

File: optimize/client/src/components/Reports/Report.js (hunk 1 above)
Comment (grlimacan) [reply optimize/client/src/components/Reports/Report.js:108]: Yes @OmranAbazid , you are right

**Review by venetrius — APPROVED** [review]
👍 Looks good to me!

**Pull request merged**